CHANGELOG.md

```diff
@@ -19,6 +19,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+## [0.6.0] - 2023-06-03
+
+### Added
+- New columnar format:
+    - Fast fields can now have any cardinality (Optional, Multivalued, restricted). In fact cardinality is now only used to format the output.
+    - Dynamic Fields are now fast fields.
+- Elasticsearch/Opensearch compatible API.
+- String fast fields now can be normalized.
+- Various parameters of object storages can now be configured.
+- The ingest API makes it possible to force a commit
+- Ability to parse non-JSON data using VRL to extract some structure from documents.
+- Object storage can now use the .
```

Comment:
"." is a cryptic feature. Do you mean "Virtual-hosted–style requests"?

Reply:
>.< yes that's what I meant.